ui/components/app/nft-details/nft-details.js

```diff
@@ -145,7 +145,6 @@ export default function NftDetails({ nft }) {
       <Box
         display={Display.Flex}
         width={inPopUp ? BlockSize.Full : BlockSize.Half}
-        margin={inPopUp ? [4, 0] : null}
```

Comment:
@NidhiKJha I removed this and added the marginTop in the other change to make it aligned. I have tested it, it is aligned for me.

Reply:
Aah! My bad. Yes, this should fix this. Thanks @ducksblock. Please fix the lint issues to make the CI checks pass and we are good to go.

Reply:
@NidhiKJha It'd be great if you can help me fix these lint errors. As I said I am new to this and these lint errors are so annoying haha

Reply:
@NidhiKJha I have fixed the lint issues, kindly review it. Thanks.